lightning-network-book.tex

```diff
@@ -83,58 +83,58 @@
 \url{https://en.wikipedia.org/w/index.php?title=Cryptographic_hash_function&oldid=868055371}}}
 
 \newglossaryentry{htlc}{name={Hashed Time Lock Contract},
-    description={Additional output in the commitment transactions, which can be claimed by one node if a secret preimage to a hash is provided within the time lock after the commitment transaction is being mined by the Bitcoin network.
+    description={Additional output in the commitment transactions, which can be claimed by one node if a secret preimage to a hash is provided within the time lock after the commitment transaction has been mined by the Bitcoin network.
       These contracts are commonly referred to as HTLCs.}}
 \newglossaryentry{payment}{name={Payment},
-  description={A Payment occurs if we transfer Bitcoin within the Lightning Network.
+  description={A payment occurs if we transfer Bitcoin within the Lightning Network.
     Payments are generally not seen on the blockchain.
-    The recipient initiates a Payment by creating an invoice.
-    The invoice includes a Payment Hash which is the Hash of a secret preimage.
-    This Payment Hash is used by the Hashed Time Lock Contracts during the routing process.}}
+    The recipient initiates a payment by creating an invoice.
+    The invoice includes a payment Hash which is the hash of a secret preimage.
```

Comment:
why Hash still with a capital H? I am really confused about capitalization anyway. I understand in English nouns usually have small letters. However terms in the glossary should have capitalized letters. that is why I also used capital letters within the text. that seemed to be wrong though : (

Reply:
Sorry, I missed that H. I'm sure that makes things more confusing.

Throughout the glossary, there was mixed use of capitals (e.g. Payment Hash in one section and payment hash in another). So I wanted to make it consistent, and chose lower case since I think that's more appropriate in general. If you look at Mastering Bitcoin, for example, most glossary entries use lowercase.

It would also be difficult to try to remember every item that has a glossary entry, so that you consistently capitlize (this is another reason for lower case). 

There were a few things that I left capitalized like Hashed Time Locked Contracts and Revocable Sequence Maturity Contracts since they are commonly written that way. In general though, capitals should be reserved for proper nouns (names like Rene, Bitcoin, Europe, etc).

I also left a couple that I was unsure of like SPHINX Mix Format and Onion Routing. I wasn't sure if they were proper nouns or not, but I think we might want to also use 'mix format' and 'onion routing' for consistency.